src/control/RecentManager.cpp

```diff
@@ -101,6 +101,12 @@ auto RecentManager::sortRecentsEntries(GtkRecentInfo* a, GtkRecentInfo* b) -> gi
     return tp_a != tp_b ? (tp_a < tp_b ? 1 : -1) : 0;
 }
 
+auto RecentManager::getMostRecent() -> GtkRecentInfo * {
```

Comment:
`clang-format`'s complaining about this line.
See https://dev.azure.com/xournalpp/xournalpp/_build/results?buildId=12744&view=logs&j=8af2c752-b8c3-5474-5b0a-7b1f58b7bf90&t=0928bceb-a691-5a8f-98f6-852338e7281a

Aside from that, this PR looks good to me!